src/actions/pause.js

```diff
@@ -24,13 +24,15 @@ type CommandType = { type: string };
  * @static
  */
 export function resumed() {
-  return ({ dispatch, client }: ThunkArgs) => {
+  return ({ dispatch, client, getState }: ThunkArgs) => {
     dispatch({
       type: "RESUME",
       value: undefined
     });
 
-    dispatch(evaluateExpressions(null));
+    if (!isStepping(getState())) {
+      dispatch(evaluateExpressions(null));
```

Comment:
it would be nice to have a test here too:


```js
+ describe("pause", () => {
+   it("should set and clear the command", async () => {
+     const client = { evaluate: jest.fn() };
+     const { dispatch, getState } = createStore(client);
+     dispatch(actions.stepIn());
+     await dispatch(actions.resumed());
+     expect(client.evaluate.calls.length).toEqual(0)
+   });
+ });
  

```